src/util-exception-policy.c

```diff
@@ -303,6 +220,76 @@ enum ExceptionPolicy ExceptionPolicyParse(const char *option, const bool support
     return policy;
 }
 
+/**
+ * \brief Evaluate whether the provided exception policy configuration
+ * setting is valid in the given midstream policy scenario, and decide on the
+ * final exception policy value based on that.
+ */
```

Comment:
This is still (also) a WIP, subject to refining - and possibly re-think how will this logic work, if we will take advantage of the existing `ExceptionPolicyParse` or not.